src/tribler/core/utilities/tracker_utils.py

```diff
@@ -116,11 +121,14 @@ def parse_tracker_url(tracker_url):
     if tracker_url.lower().startswith('http://') and ':80/' in tracker_url:
         tracker_url = tracker_url.replace(':80/', '/', 1)
 
+    if tracker_url.lower().startswith('https://') and ':443/' in tracker_url:
+        tracker_url = tracker_url.replace(':443/', '/', 1)
```

Comment:
Why is this code necessary? Can you provide an example of the incorrect URL that this code fixes? (I suspect the fixed URL is not entirely correct and can include an excess `/`, but may be wrong on that)